### DISCUSSION TITLE
How to set currentTime from parrent page

### DISCUSSION THREAD

**Discussion by FlakM** [podverse/podverse-web#1071] [Q&A]
Hello! I am working on a project where I need to add links that direct users to a specific moment in a podcast on the Jupiter Broadcasting website. 

I understand that the web player is embedded using an iframe and due to the cross-origin policy, I am unable to access the audio element to set the time. I have not found any event listeners that would allow me to communicate with the parent page. Is there any other way to achieve this functionality?

Also obligatory message: thank you for your awesome work! <3

**Comment by mitchdowney** [comment]
Hello @FlakM! Hmm it sounds like something a simple "start time" URL parameter could handle? I don't think we have URL param support like that yet (unless I forgot about it). I went ahead and created this Github issue for the work needed: https://github.com/podverse/podverse-web/issues/1072

I'm swamped with major feature work right now (updating our mobile app audio and video libraries), but this seems like low-hanging fruit so hopefully I can get it done within the next week.

I guess an alternative would be to allow sending an event (or is it message?) to the iframe from the parent window. If you need something more like that, could you create a Github issue describing the use case and what's needed?

Thanks for your kind feedback 😊

**Comment by FlakM** [comment]
Thanks for the response. I was imagining having a [postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) API to control player state. If I understand correctly having url param would have solved my original problem but it might result in unnecessary load on the server side.

**Reply by mitchdowney** [reply]
Ok cool. I should be able to add both. It should be pretty simple, although I'm swamped with work until possibly next weekend.

**Reply by FlakM** [reply]
Wow that's awesome. There is absolutely no pressure on my side, just wanted to make sure that I'm not missing some obvious solution.
